pensieve/config.py

```diff
@@ -26,31 +26,90 @@
 import jinja2
 import mozanalysis.metrics
 import mozanalysis.metrics.desktop
+import pandas
 
-from pensieve.analysis import AnalysisPeriod
+from . import AnalysisPeriod
+from pensieve.statistics import Statistic, BootstrapMean, StatisticResultCollection
 import pensieve.experimenter
 
 
-# TODO: these should be objects that pair a metric with a default statistical treatment.
+@attr.s(auto_attribs=True)
+class MetricWithTreatment:
```

Comment:
This might deserve a better name?

Reply:
I've been idly bikeshedding this for a few days and I'm not in love with anything yet but I'll sleep on it; this is as good as anything I've thought of

Reply:
The best I've come up with is `Summary`